config/peerpods/mc-50-crio-config.yaml

```diff
@@ -2,7 +2,8 @@ apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
   labels:
-    machineconfiguration.openshift.io/role: worker
+    machineconfiguration.openshift.io/role: kata-oc
+    app: example-kataconfig
```

Comment:
Does this belong upstream? (A honest question, admittedly I'm not sure how the `app` label is used.)

Reply:
It's a placeholder. This is not used by the MCO. The `app` label is also set for sandboxed extension - https://github.com/openshift/sandboxed-containers-operator/blob/main/controllers/openshift_controller.go#L483 
I think I should change it to the peerpodconfig name